languages/de.json

```
@@ -773,6 +813,10 @@
 "DND5E.HealthFormula": "TP-Formel",
 "DND5E.Height": "Größe",
 "DND5E.HitDice": "Trefferwürfel",
+"DND5E.HitDiceAutoSpend": {
+  "Label": "Automatische  Trefferwürfel verwenden",
+  "Hint": "Geben Sie automatisch Trefferwürfel aus, bis sie aufgebraucht sind oder die Lebenspunkte voll sind."
```

```suggestion
  "Hint": "Automatisch Trefferwürfel ausgeben, bis sie aufgebraucht sind oder die Lebenspunkte voll sind."
```
I tried to avoid `Sie`/`Du` in most translations, if possible :)